experiment_scheduler/submitter/__init__.py

```diff
@@ -26,9 +27,12 @@ def _run_as_default_process(command, target):
         args=command, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
     )
     print(f"now {target} is running")
+    decode_type = "utf-8"
+    if platform.system() == "Windows":
+        decode_type = "cp949"
     try:
         for line in iter(task.stdout.readline, b''):
-            print(line.decode("utf-8"))
+            print(line.decode(decode_type))
```

Comment:
Print would better if replaced by logger.

Reply:
This part print each servers' stdout. I think `print` is appropriate.